### ISSUE TITLE
Bug: it does not appear to export GA

### ISSUE THREAD

**Issue by csellis** [beyonk-adventures/svelte-google-analytics#13]
## Problem
Was using this plugin with the GDPR consent plugin. I started this one with it disabled and tried importing `ga`.

## Expected
- Export of `ga` so I can start GA
- 
https://github.com/beyonk-adventures/svelte-google-analytics/blob/6ce00b76c7230956413eeedfb8e3068c387e114e/src/generated-functions.js#L381

**Comment by antony** [comment]
Yeah the point is not to export `ga`, as we already build all the functions in :)
